CHANGELOG.md

```diff
@@ -53,11 +41,12 @@ Hyperledger Besu is moving its versioning scheme to [CalVer](https://calver.org/
 - [Privacy users with private transactions created using v1.3.4 or earlier](KNOWN_ISSUES.md#privacy-users-with-private-transactions-created-using-v134-or-earlier)
 - [Changes not saved to database correctly causing inconsistent private states](KNOWN_ISSUES.md#Changes-not-saved-to-database-correctly-causing-inconsistent-private-states)
```

Comment:
@lucassaldanha and @pinges - this was fixed wasn't it?

Reply:
Yes, that seems to be fixed by our changes.